integration-tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@slack/bolt-integration-tests",
+  "private": true,
+  "description": "Integration tests for Bolt",
+  "author": "Slack Technologies, Inc.",
+  "license": "MIT",
+  "scripts": {
+    "test": "npm run test:typechecker",
+    "test:typechecker": "dtslint types"
+  },
+  "dependencies": {
+    "@slack/bolt": "^1.4.1",
+    "@types/node": "^8.10.43"
+  },
+  "devDependencies": {
+    "dtslint": "^0.5.4",
+    "typescript": "^3.3.3333"
```

Comment:
Can we set this to a later version of TypeScript? the README in this directory says that it will use the most modern version of TS available.

Reply:
Yup. I just copied the entire integration tests folder from `node-slack-sdk` and then made updates. I've updated the TS version now